src/Workspaces/Core/Portable/CodeFixes/CodeFixContext.cs

```diff
@@ -15,33 +16,27 @@ namespace Microsoft.CodeAnalysis.CodeFixes
     /// <summary>
     /// Context for code fixes provided by a <see cref="CodeFixProvider"/>.
     /// </summary>
-    public struct CodeFixContext : ITypeScriptCodeFixContext
+    public readonly struct CodeFixContext : ITypeScriptCodeFixContext
     {
         private readonly Document _document;
-        private readonly Project _project;
         private readonly TextSpan _span;
         private readonly ImmutableArray<Diagnostic> _diagnostics;
         private readonly CancellationToken _cancellationToken;
         private readonly Action<CodeAction, ImmutableArray<Diagnostic>> _registerCodeFix;
 
         /// <summary>
-        /// Document corresponding to the <see cref="CodeFixContext.Span"/> to fix.
+        /// Document corresponding to the <see cref="Span"/> to fix.
         /// </summary>
         public Document Document => _document;
 
         /// <summary>
-        /// Project corresponding to the diagnostics to fix.
-        /// </summary>
-        internal Project Project => _project;
```

Comment:
doesn't hurt to keep this right?  can be impl'ed as `=> Document.Project;`

(not suggesting this... just curious why we prefer to remove).

Reply:
Preferred to remove since it's only used in a few places, so it does not save much